docs/deployment.rst

```diff
@@ -51,10 +51,10 @@ But this way requires more complex configuration.
 Nginx configuration
 --------------------
 
-Here is short extraction about writing Nginx configuration file.
+Here is short example of an Nginx configuration file.
```

Comment:
```suggestion
Here is a short example of an Nginx configuration file.
```

Reply:
Oh, it got automerged before I saw this. Maybe send a follow up?